README.md

```diff
@@ -27,6 +27,9 @@ of them assume you have docker running on your local machine.
     - This module demonstrates using the OpenTelemetry Java Agent with a simple
       spring boot application. Traces, metrics, and logs are exported to a
       collector via OTLP.
+- [Spring native image telemetry with OpenTelemetry Spring Starter](spring-native)
+    - This module demonstrates using the OpenTelemetry Spring Boot starter with a
+      Graal VM native image. Traces and metrics are exported to a collector via OTLP.
```

Comment:
I see mention of logs below and the exporter set to OTLP. Does this also export logs? If so, maybe update this line?

Reply:
Log export should be available from next opentelemetry-java-instrumentation release with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8493